grails-app/assets/javascripts/measurementSetup/_createJob.js

```diff
@@ -110,23 +93,6 @@ OpenSpeedMonitor.MeasurementSetupWizard.CreateJobCard = (function () {
         validateInputs();
     }
 
-    var getExistingJobNames = function () {
-        var url = OpenSpeedMonitor.urls.getJobNames;
```

Comment:
This URL seems to be unused now. I think you can safely remove the URL (from `_jsGlobals`) and also the controller method from the `MeasurementSetupController`